litex/soc/software/common.mak

```diff
@@ -53,7 +53,7 @@ INCLUDES = -I$(SOC_DIRECTORY)/software/include/base \
 COMMONFLAGS = $(DEPFLAGS) -Os $(CPUFLAGS) -g3 -fomit-frame-pointer -Wall -fno-builtin -nostdinc -fno-stack-protector $(INCLUDES)
 CFLAGS = $(COMMONFLAGS) -fexceptions -Wstrict-prototypes -Wold-style-definition -Wmissing-prototypes
 CXXFLAGS = $(COMMONFLAGS) -std=c++11 -I$(SOC_DIRECTORY)/software/include/basec++ -fexceptions -fno-rtti -ffreestanding
-LDFLAGS = -nostdlib -nodefaultlibs -L$(BUILDINC_DIRECTORY)
+LDFLAGS = -nostdlib -nodefaultlibs -Wl,--no-dynamic-linker -Wl,--build-id=none -L$(BUILDINC_DIRECTORY)
```

Comment:
Out of interest, when were `--no-dynamic-linker` and `--build-id` options added to `ld`? (Looks like 2015 - https://reviews.llvm.org/D30258 ?)

Reply:
Yes, it looks like Sept 2015:

 https://sourceware.org/git/?p=binutils-gdb.git;a=commit;h=9b8b325a1f4cdaf235e7d803849dde6ededec865
 
This means it first appeared in binutils 2.26, which was released in Jan 2016.